function-data-migration/src/main/java/com/future/function/data/migration/change/log/DataMigration_020.java

```diff
@@ -0,0 +1,158 @@
+package com.future.function.data.migration.change.log;
+
+import com.future.function.common.enumeration.core.Role;
+import com.future.function.model.entity.feature.core.Access;
+import com.future.function.model.util.constant.DocumentName;
+import com.future.function.model.util.constant.FieldName;
+import com.github.mongobee.changeset.ChangeLog;
+import com.github.mongobee.changeset.ChangeSet;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ChangeLog(order = "020")
+public class DataMigration_020 {
+
+    @ChangeSet(author = "david",
+            id = "addUrlRegexForJudgingList",
+            order = "0001")
+    public void addUrlRegexForJudgingList(
+            MongoTemplate mongoTemplate
+    ) {
+
+        String urlRegex = "^\\/final-judging(\\/)?$";
```

Comment:
Need to include query params? If so maybe `urlRegex` can be changed to `^\\/final-judging(\\/)?(\\?.*)?$`

Reply:
cc @DavidWilliamK